src/test/java/LocationsUtilityTest.java

```diff
@@ -99,23 +82,33 @@ public void getIncorrectlyEnclosedLocation() {
     // [].
     Assert.assertEquals(
         ImmutableList.of(),
-        LocationsUtility.getLocations(PREFIX, TASKS_WITH_ONE_LOCATION_ENCLOSED_INCORRECTLY));
+        LocationsUtility.getLocations(
+            PREFIX, ImmutableList.of(TASK_WITH_ONE_LOCATION_ENCLOSED_INCORRECTLY)));
   }
 
   @Test
   public void getEmptyLocation() {
     // Obtain location in the task notes of one task with one location with [Location: ] tag but
     // nothing inside of it.
     Assert.assertEquals(
-        ImmutableList.of(""), LocationsUtility.getLocations(PREFIX, TASKS_WITH_EMPTY_LOCATION));
+        ImmutableList.of(),
+        LocationsUtility.getLocations(PREFIX, ImmutableList.of(TASK_WITH_EMPTY_LOCATION)));
   }
 
   @Test
   public void getLocationAllTasks() {
     // Obtain location in the task notes of five tasks each with either one, empty or no location as
     // noted in the individual tests above.
```

Comment:
It's not clear what is the point of this test. You've already tested the tasks that produce no location, and the tasks that produce `LOCATION`. I think this test case would make more sense if you had two tasks that produced different locations, and you check that both are returned. 

(You can keep the other tasks which produce duplicate locations or no locations at all. I am specifically pointing out the need to test returning multiple locations.)

Reply:
Fair point! Mindset starting to change now that there's so many pending PRs and so little time. I will be more careful.

Reply:
Fair point! I will be more careful with changing other stuff based on review changes. Thanks for pointing it out!